{
  try {
    jobManagerSystem.actorOf(Props.create(FailureReturningActor.class),JobManager.JOB_MANAGER_NAME());
    Client out=new Client(config);
    try {
      out.runDetached(program.getPlanWithJars(),1);
      fail("This should fail with an exception");
    }
 catch (    ProgramInvocationException e) {
    }
catch (    Exception e) {
      fail("wrong exception " + e);
    }
  }
 catch (  Exception e) {
    e.printStackTrace();
    fail(e.getMessage());
  }
}
